docs/matrix.rst

```diff
@@ -90,11 +90,11 @@ As a temporary local variable
 Accessing components
 --------------------
 
-As global fields of vectors
-+++++++++++++++++++++++++++
+As global vector fields
```

Comment:
?? Maybe it's not your fault, let's fix by-the-way:
```suggestion
As global matrix fields
```

Reply:
Very nice! I checked my local repo and shows that this has been replaced... strange.